rotkehlchen/types.py

```diff
@@ -1230,6 +1230,10 @@ class ProtocolsWithCache(SerializableEnumNameMixin):
     CONVEX = auto()
     GEARBOX = auto()
     SPARK = auto()
+    # TODO: This should be removed once https://github.com/rotki/rotki/issues/9302 implemented
+    ETH_WITHDRAWALS = auto()
+    # TODO: This should be removed once https://github.com/rotki/rotki/issues/9302 implemented
+    ETH_BLOCKS = auto()
```

Comment:
```suggestion
    # TODO: ETH_WITHDRAWALS and ETH_BLOCKS should be removed once https://github.com/rotki/rotki/issues/9302 implemented
    ETH_WITHDRAWALS = auto()
    ETH_BLOCKS = auto()
```
no need to have the same comment twice.